src/components/Person.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Person as PersonInterface } from '../types';
+import { Link, useSearchParams } from 'react-router-dom';
+
+type Props = {
+  data: PersonInterface;
+  param: string | undefined;
+};
+
+export const Person: React.FC<Props> = ({ data, param }) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  } = data;
+  const isFather = fatherName ? fatherName : '-';
+  const isMother = motherName ? motherName : '-';
```

Comment:
The variables `isFather` and `isMother` are used to store the father's and mother's names or a placeholder. Consider renaming these variables to `fatherDisplayName` and `motherDisplayName` for better clarity and readability.